templates/test/dev/cluster-template-custom-builds-machine-pool.yaml

```diff
@@ -114,13 +114,11 @@ spec:
         set -o pipefail
         set -o errexit
 
-        systemctl stop kubelet
         declare -a BINARIES=("kubeadm" "kubectl" "kubelet")
         for BINARY in "$${BINARIES[@]}"; do
           echo "* installing package: $${BINARY} ${KUBE_GIT_VERSION}"
           curl --retry 10 --retry-delay 5 "https://${AZURE_STORAGE_ACCOUNT}.blob.core.windows.net/${JOB_NAME}/${KUBE_GIT_VERSION}/bin/linux/amd64/$${BINARY}" --output "/usr/bin/$${BINARY}"
         done
-        systemctl restart kubelet
```

Comment:
We're downloading a binary outside of the package manager here–why don't we want to restart the service that calls it?

Reply:
@mboersma doesn't kubeadm start kubelet though? The image-builder just enables it but state is [stopped](https://github.com/kubernetes-sigs/image-builder/blob/master/images/capi/ansible/roles/kubernetes/tasks/main.yml#L58). Since this is running in preKubeadmCommands we shouldn't be restarting a service that hasn't started yet.